__tests__/app.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest');
+const {app, server} = require('../app.js');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const data = require('../db/data/test-data/index.js')
+
+beforeEach(() => {
+    return seed(data);
+});
+
+afterAll(() => {
+    return db.end().then(() => {return server.close()});
+})
+
+describe('Invalid endpoint requests', () => {
+    test('404: should return not found in response to get request to non-existent endpoint', () => {
+        return request(app)
+            .get('/api/does-not-exist')
+            .expect(404)
+    });    
+});
```

Comment:
Although not specific to this endpoint its nice to see this added, but there should also have an expect statement here for what we want the message to be on the response object.